APP-MANAGER

```diff
@@ -10,6 +14,25 @@ CLI=$(basename "$0")
 arch="$HOSTTYPE"
 currentuser="$USER"
 
+# Determine sytem package manager
```

Comment:
this is not necessary, it can be a potential problem in the future. at least if user wants am to manage the installation if not exit with error msg (please install X using you package manager)